source/common/http/utility.cc

```diff
@@ -756,6 +757,62 @@ void Utility::sendLocalReply(const bool& is_reset, const EncodeFunctions& encode
   encodeLocalReply(is_reset, std::move(prepared_local_reply));
 }
 
+bool Utility::remoteAddressIsTrustedProxy(
+    const Envoy::Network::Address::InstanceConstSharedPtr& remote,
+    const std::vector<Network::Address::CidrRange> trusted_cidrs) {
+  for (const auto& cidr : trusted_cidrs) {
+    if (cidr.isInRange(*remote.get())) {
+      return true;
+    }
+  }
+  return false;
+}
+
+Utility::GetLastAddressFromXffInfo Utility::getLastNonTrustedAddressFromXFF(
+    const Http::RequestHeaderMap& request_headers,
+    const std::vector<Network::Address::CidrRange> trusted_cidrs) {
+  const auto xff_header = request_headers.getForwardedForValue();
+  static constexpr absl::string_view separator(",");
+  static constexpr size_t MAX_ALLOWED_XFF_ENTRIES = 20;
+
+  const auto xff_entries = StringUtil::splitToken(xff_header, separator, false, true);
```

Comment:
which type is used here to store the splitted tokens? Seems `auto` is not appropriate here. Maybe you should specify a type here. InlineVector<absl::string_view, 8> will be recommend.

Reply:
Updated. I used `std::vector<absl::string_view>` as this seemed to be the style elsewhere and I couldn't find any reference of `InlineVector`.

Reply:
`absl::InlinedVector` provide better performance when we only has limited hops (in most cases?). And sorry for the spelling error.

Reply:
Thanks for the explanation. I've updated to use InlinedVector now.